sdk/identity/pom.xml

```diff
@@ -12,5 +12,6 @@
     <!-- NOTE: Commenting out becuase we are now binary composed.
     -->
     <module>azure-identity</module>
+    <module>perf-test</module>
```

Comment:
Use the artifact id here: `azure-identity-perf`

Reply:
Updated to azure-identity-perf